pkg/pipeline/input/sdk/appwriter.go

```diff
@@ -192,29 +192,26 @@ func (w *appWriter) start() {
 				return
 			}
 
-			// when a track is killed, one 8x8 frame is sent by the SFU and needs to be ignored
-			if pkt != nil {
```

Comment:
we no longer need this?

Reply:
Nope. This was an attempt to get around the final packet and worked for killing a video track, but doesn't work for screen share. The screen share fix of allowing the error and sending an eos to that appwriter also works on regular video tracks